### PR TITLE
Sending EDR to Analytics and calling Search for movie title for getTopRated

### PR DESCRIPTION
**Pull request by kedarkhetia**
Test URL: https://hist-fav-checkout-beta.herokuapp.com/user?userId=1

Another way of checking is, pull the code and add print statement in AnalyticsFilter doFilter method and run the code locally.

================== Note ==================

There will be changes in EDR structure based on the discussion with Analytics team.

### CODE REVIEW

File: src/cs/usfca/edu/histfavcheckout/filters/ExcludeFilters.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ExcludeFilters {
+	@Bean
+	public FilterRegistrationBean<AuthenticationFilter> filter() {
+		FilterRegistrationBean<AuthenticationFilter> bean = new FilterRegistrationBean<>();
+		bean.setFilter(new AuthenticationFilter());
+//		String[] urlPatterns = {"/rateMovie/", "/totalFavesAndCheckouts/", "/checkedOutMovies/",
```
Comment (mosopeogundipe) [inline src/cs/usfca/edu/histfavcheckout/filters/ExcludeFilters.java]: Is there any need to keep this commented section?

File: src/cs/usfca/edu/histfavcheckout/filters/ExcludeFilters.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		bean.addUrlPatterns("/rateMovie", "/totalFavesAndCheckouts", "/checkedOutMovies", 
+				"/returnMovies", "/checkOutMovies", "/user");  // or use setUrlPatterns()
+		return bean;
+	}
+	
+	@Bean
+	public FilterRegistrationBean<AnalyticsFilter> analyticsFilter() {
+		FilterRegistrationBean<AnalyticsFilter> bean = new FilterRegistrationBean<>();
+		bean.setFilter(new AnalyticsFilter());
+//		String[] urlPatterns = {"/rateMovie/", "/totalFavesAndCheckouts/", "/checkedOutMovies/",
```
Comment (mosopeogundipe) [inline src/cs/usfca/edu/histfavcheckout/filters/ExcludeFilters.java]: Is there any need to keep this commented section?

**Review by mosopeogundipe — APPROVED** [review]
I left just two comments. About maybe removing some commented portions of code. It's not important, really

File: src/cs/usfca/edu/histfavcheckout/filters/ExcludeFilters.java (hunk 1 above)
Comment (kedarkhetia) [reply src/cs/usfca/edu/histfavcheckout/filters/ExcludeFilters.java]: No, I will remove it.

File: src/cs/usfca/edu/histfavcheckout/filters/ExcludeFilters.java (hunk 2 above)
Comment (kedarkhetia) [reply src/cs/usfca/edu/histfavcheckout/filters/ExcludeFilters.java]: No, I will remove it.

**Pull request merged**